### PR TITLE
feat(android): viewShadowColor parity

### PR DESCRIPTION
**Pull request by m1ga**
Adds parity for `viewShadowColor`.

Example:

```js
const window = Ti.UI.createWindow({});
const v1 = Ti.UI.createView({
	width: 100,
	height: 100,
	left: 50,
	backgroundColor: "#fff",
	elevation: 40,
})
const v2 = Ti.UI.createView({
	width: 100,
	height: 100,
	right: 50,
	backgroundColor: "#fff",
	elevation: 40,
	viewShadowColor: "#00f"
})

v1.addEventListener("click", e => {
	v1.viewShadowColor = "yellow";
});

v2.addEventListener("click", e => {
	v2.viewShadowColor = "red"
})

window.add([v1, v2]);
window.open();
```
open the app and click on the views.

Screenshot_20220522-151400

You can do it already with the current SDK using your theme:
```xml
    <style name="AppTheme" parent="Theme.Titanium.DayNight.Solid.NoTitleBar">
        <item name="android:ambientShadowAlpha">0.5</item>
        <item name="android:spotShadowAlpha">0.5</item>

        <item name="android:outlineAmbientShadowColor">@color/defaultOutlineAmbientShadowColor</item>
        <item name="android:outlineSpotShadowColor">@color/defaultOutlineSpotShadowColor</item>
    </style>
    
    <color name="defaultOutlineAmbientShadowColor">#ff0000</color>
    <color name="defaultOutlineSpotShadowColor">#ff0000</color>
```

### CODE REVIEW

File: android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java [hunk 1]
```diff
@@ -951,6 +951,17 @@ public void propertyChanged(String key, Object oldValue, Object newValue, KrollP
...
+		} else if (key.equals(TiC.PROPERTY_VIEW_SHADOW_COLOR)) {
+			if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+				if (nativeView != null) {
+					nativeView.setOutlineAmbientShadowColor(TiConvert.toColor(TiConvert.toString(newValue),
+						TiApplication.getAppCurrentActivity()));
+					nativeView.setOutlineSpotShadowColor(TiConvert.toColor(TiConvert.toString(newValue),
+						TiApplication.getAppCurrentActivity()));
+				}
+			} else {
+				Log.w(TAG, "only available for Android P+");
```
Comment (hansemannn) [inline android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java:963]: Suggestion for a better phrasing:
> Setting the "viewShadowColor" property  requires Android P or later

File: android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java [hunk 2]
```diff
@@ -1105,6 +1116,19 @@ public void processProperties(KrollDict d)
...
+		if (d.containsKey(TiC.PROPERTY_VIEW_SHADOW_COLOR) && !nativeViewNull) {
+			if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+				nativeView.setOutlineAmbientShadowColor(
+					TiConvert.toColor(TiConvert.toString(d, TiC.PROPERTY_VIEW_SHADOW_COLOR),
+						TiApplication.getAppCurrentActivity()));
+				nativeView.setOutlineSpotShadowColor(
+					TiConvert.toColor(TiConvert.toString(d, TiC.PROPERTY_VIEW_SHADOW_COLOR),
+						TiApplication.getAppCurrentActivity()));
+			} else {
+				Log.w(TAG, "only available for Android P+");
```
Comment (hansemannn) [inline android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java:1128]: Same here

File: android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java (hunk 1 above)
Comment (m1ga) [reply android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java]: sounds good :+1:

**Review by hansemannn — APPROVED** [review]
Looks good! Can be merged once master is bumped to 11.1.0 (will do today)

**Comment by caspahouzer** [comment]
As Hans already bumped the master to 11.1.0 and he approved this PR, LGTM

**Pull request merged**